x/ccv/provider/client/legacy_proposals.go

```diff
@@ -43,6 +43,7 @@ type ConsumerAdditionProposalJSON struct {
 	Denylist           []string `json:"denylist"`
 	MinStake           uint64   `json:"min_stake"`
 	AllowInactiveVals  bool     `json:"allow_inactive_vals"`
+	Prioritylist       []string `json:"prioritylist"`
```

Comment:
Do not update legacy proposals as we no longer need them. Actually, why do we even have this file?

Reply:
Priority list is now removed from legacy_proposals.go.

Reply:
@MSalopek do you know why do we still need this file? cc @bermuell

Reply:
My mistake, the priority list is needed in legacy proposal for e2e test.

Reply:
Why do we still legacy props on our e2e tests? Can we get rid of these? Not something we need to do in this PR though.

Reply:
Don't remove them because you'll break governance on gaia. They have to exist so old Txs can be displayed and handled properly. Otherwise you will not be able to fetch an old block or check an old gov proposal because the gov type is missing.

Please don't remove it.